.golangci.yml

```diff
@@ -41,6 +41,10 @@ linters-settings:
       - G306
       - G402
       - G404
+  misspell:
+    ignore-words:
+      - transferer
+      - transferers
```

Comment:
nit on spelling...

transfer is one of those weird words where the noun doesn't look right when spelled correctly because you are used to the verb spelling...have to add an extra r when going from transfer to transferrer(s)